webrtc/videoroom/assets/src/room.ts

```diff
@@ -66,6 +66,7 @@ export class Room {
           addVideoElement(peer.id, peer.metadata.displayName, false);
         },
         onPeerLeft: (peer) => {
+          console.log("Peer left");
```

Comment:
We don't have any other logs, about peer joining etc. so maybe let's be consistent and remove it?